tasks/sign-index-image/README.md

```diff
@@ -27,6 +27,9 @@ data:
         configMapName: <configmap name>
 ```
 
+## Changes in 3.0.0
+- This task now requires a list of digests to use in the signing request via the parameter `manifestListDigests`
```

Comment:
Please also say that the other parameter was removed.

Reply:
fixed